src/components/appNavigation/TabsView.js

```diff
@@ -117,34 +115,35 @@ const InviteButton = ({ onPress, style }) => (
   </>
 )*/
 
-const EmptySpaceComponent = ({ style }) => (
-  <>
-    <View style={style} />
-    <Appbar.Content />
-  </>
-)
+// const EmptySpaceComponent = ({ style }) => (
+//   <>
+//     <View style={style} />
+//     <Appbar.Content />
+//   </>
+// )
 
 const TabsView = ({ navigation }) => {
   const { slideToggle } = useSideMenu()
-  const [token, setToken] = useState(isIOS ? undefined : true)
-  const [marketToken, setMarketToken] = useState(isIOS ? undefined : true)
 
-  const fetchTokens = useCallback(async () => {
-    let _token = await userStorage.getProfileFieldValue('loginToken')
+  // const [token, setToken] = useState(isIOS ? undefined : true)
```

Comment:
why do not remove this totally ? as we won't have etoro & market anymore ?
even we would need that we could always restore from the git history

Reply:
its removed in the other PR